PxtlCa.XmlCommentMarkDownGenerator/XmlToMarkDown.cs

```diff
@@ -63,7 +61,7 @@ public static string ToMarkDown(this XNode node, string assemblyName = null)
                     name = anchor ? "seeAnchor" : "seePage";
                 }
                 //treat first Param element separately to add table headers.
-                if (name == "param"
+                if (name.EndsWith("param")
```

Comment:
I'm not 100% fond of this approach, but this codebase is already just a big mess of string parsing with delusions of grandeur so I'll go with it. At some point all these strings will have to get wrapped up in objects or something and this will have to get split into some sensible OOP method on Param and TypeParam types, but that's not today.